src/stylesheets/blocks/_header.scss

```diff
@@ -1 +1,143 @@
+.header {
+  background-color: color(mint-green);
+  width: 100%;
+}
+
+.header-wrapper {
+  display: block;
+  margin: 0 auto;
+  max-width: rem(680);
+  position: relative;
+}
+
+.header__background-img {
+  background-repeat: no-repeat; 
+  background-size: cover;
+  display: inline-block;
+  position: absolute;
+}
+
+.img-1 {
+  background-image: url('../../images/Pill_Orange_Title-01.svg');
```

Comment:
nbd but all lowercase and dash-separated filenames are standard, at least in my experience

Reply:
gotcha. these were the file names in figma (not my preference either), but i didn't modify for this project so that it would be easier to identify what was what between the figma design and my build

Reply:
👍